engine/resources/net.go

```diff
@@ -37,6 +37,7 @@ import (
 
 	multierr "github.com/hashicorp/go-multierror"
 	errwrap "github.com/pkg/errors"
+	"github.com/purpleidea/mgmt/lib/socketset"
```

Comment:
The lib package is meant for general core "mgmt as a lib" stuff (sorry i suck at naming) so how about we put socketset into: `github.com/purpleidea/mgmt/util/socketset` instead?